projects/fmcadc2/common/fmcadc2_bd.tcl

```diff
@@ -21,7 +28,7 @@ adi_tpl_jesd204_rx_create axi_ad9625_core $RX_NUM_OF_LANES \
 adi_axi_jesd204_rx_create axi_ad9625_jesd 8
```

Comment:
jesd instatiation is also a function of the number of lanes.

Reply:
Changed jesd instantiation to be a function of the number of lanes